configure.ac

```
@@ -855,7 +855,7 @@ if test "x$enable_backtrace" != xno; then
    AC_DEFINE([HAVE_BACKTRACE], [1], [Define if the backtrace feature is enabled])
 fi
 
-if test "x$enable_backtrace" = xunwind-ptrace; then
+if test "x$enable_backtrace" = xunwind-ptrace -a "x$my_htop_platform" = xlinux; then
```

Things can be concatenated, actually.

```suggestion
if test "X${enable_backtrace}X$my_htop_platform" = Xunwind-ptraceXlinux; then
```

Alternatively use this (Autoconf manual does not recommend the `-a` option in `test` and instead suggests this to be more portable):

```suggestion
if test "x$enable_backtrace" = xunwind-ptrace && test "x$my_htop_platform" = xlinux; then
```

---

I prefer the second one. Good catch!